content/blog/2022/2022-09-25-node-urlShortener.md

```diff
@@ -0,0 +1,494 @@
+---
+title: "Building a Url Shortener With Node.Js"
+categories: ["Node"]
+date: 2022-09-25 00:00:00 +1100
+authors: ["ajibade"]
+description: "Building A MERN stack Url Shortener Application"
+image: images/stock/0043-calendar-1200x628-branded.jpg
+url: node-url-shortener
+---
+
+
+The best URL shortening service depends on your needs and what you want it to do. There are a lot of simple, fast, and free URL shortening services. Some provide detailed information about who is clicking your links, the number of times they are clicked and even options to add a [call to action](https://en.wikipedia.org/wiki/Call_to_action_(marketing)) to links. URL shorteners like [bitly](https://bitly.com/) and [cutt.ly](https://cutt.ly/) are popular examples.
+
+This tutorial will walk you through how to develop a URL shortener application using the `MERN stack`. Developing `REST APIs` with `Node.js` and `MongoDB` and then integrating them into `React.js` frontend applications. This is a common coding interview question and an excellent practice project for new `Node.js` and `React` developers.
```

Comment:
Please only use `code highlighting` for variable names or code snippets. Not for words like MERN stack or REST APIs.